src/Raven.Studio/typescript/widgets/virtualGrid/textColumn.ts

```diff
@@ -0,0 +1,25 @@
+﻿import virtualColumn = require("widgets/virtualGrid/virtualColumn");
+
+/**
+ * Virtual grid column that renders text in its cells.
+ */
+class textColumn implements virtualColumn {
+    readonly cellClass = "text-cell";
+
+    constructor(
+        public dataMemberName: string,
+        public display: string,
+        public width: string) {
+    }
+
+    renderCell(item: Object, isSelected: boolean): string {
+        const cellValue = (item as any)[this.dataMemberName];
+        if (cellValue != null) {
```

Comment:
I considered that as well, but also want to filter out undefined, in case the row data doesn't have that cell property defined. I could have written if (cellValue !== null && cellValue !== undefined), but (cellValue != null) covers both cases. One of those rare times JavaScript's loose equality (double equals) is actually useful. :-)